### PR TITLE
[WEB-69] [WEB-70] [WEB-71] The rest of EventsV1

### PR DESCRIPTION
**Pull request by UncleClapton**

### CODE REVIEW

File: components/Groups/GroupEventCard.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+// import Button from '../Button'
+import Component from '../Component'
+import Markdown from '../Markdown'
+
+
+
+
+class GroupEventCard extends Component {
+  async componentDidMount () {
+    /*const {
```
Comment (UncleClapton) [inline components/Groups/GroupEventCard.js]: Commented out due to API-side things not being ready.

File: components/Groups/GroupEventEditDialog.js [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+    } = this.props
+
+    const {
+      changes,
+      submitting,
+    } = this.state
+
+
+    console.log(changes.startTime)
+    console.log(event.attributes.start_time)
```
Comment (trezy) [inline components/Groups/GroupEventEditDialog.js]: HEY. LISTEN.

File: components/Groups/GroupEventEditDialog.js [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+
+    const description = typeof changes.description === 'string' ? changes.description : event.attributes.description
+    const endTime = typeof changes.endTime === 'object' ? changes.endTime : new Date(event.attributes.end_time)
+    const game = typeof changes.game === 'object' ? changes.game : (currentGame || '')
+    const location = typeof changes.location === 'object' ? changes.location : event.attributes.location
+    const startTime = typeof changes.startTime === 'object' ? changes.startTime : new Date(event.attributes.start_time)
+    const title = typeof changes.title === 'string' ? changes.title : event.attributes.title
+
+    console.log(startTime)
```
Comment (trezy) [inline components/Groups/GroupEventEditDialog.js]: HEY. LISTEN.

Comment (UncleClapton) [reply components/Groups/GroupEventEditDialog.js]: >_< OH NOOO

File: pages/groups/group.js [hunk 4]
```diff
@@ -447,8 +487,8 @@ class GroupProfile extends Component {
               category="Groups"
               className="details"
               defaultTab={initialTab}
-              onSelect={tabId => {
-                const route = `${window.location.pathname.replace(/\/(details|join-requests|members|settings)/, '')}/${tabId}`
+              onSelect={async tabId => {
```
Comment (UncleClapton) [inline pages/groups/group.js]: why the heck is this here

**Pull request merged**